roles/openshift_node/tasks/main.yml

```diff
@@ -40,7 +40,7 @@
   when: not openshift.common.is_containerized | bool
 
 - name: Install sdn-ovs package
-  action: "{{ ansible_pkg_mgr }} name={{ openshift.common.service_type }}-sdn-ovs{{ openshift_version | oo_image_tag_to_rpm_version(include_dash=True) }} state=present"
+  action: "{{ ansible_pkg_mgr }} name={{ openshift.common.service_type }}-sdn-ovs{{ openshift_version | default('') | oo_image_tag_to_rpm_version(include_dash=True) }} state=present"
```

Comment:
I would expect the default for openshift_version to be handled in the defaults/main.yml for this role.